### PR TITLE
Automated virtual env setup for development with a Makefile

### PR DESCRIPTION
**Pull request by basimr**
This pull request is for #4. By adding a Makefile that automates environment setup, it'll be easier for the both of us to maintain the same development environment.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+prepare-venv: clean
+	virtualenv --no-site-packages env
+	env/bin/pip install -r requirements.txt
+
+start-engine:
+	env/bin/python env/bin/gunicorn --reload engine.api:app
+
+clean:
+	rm -rf ./env
```
Comment (ali-ramadhan) [inline Makefile]: Add -v (--verbose) flag?

File: engine/api.py [hunk 2]
```diff
@@ -4,8 +4,8 @@
 import logging
 
 from problems.problem1 import Problem1
-from runners.python_runner import PythonRunner
-import util
+from engine.runners.python_runner import PythonRunner
+import engine.util as util
 
 # Config applies to all other loggers
 logging.basicConfig(format='[%(asctime)s] %(name)s:%(levelname)s: %(message)s', level=logging.DEBUG)
```
Comment (ali-ramadhan) [inline engine/api.py]: Might use a different format later.

File: Makefile (hunk 1 above)
Comment (basimr) [inline Makefile]: Good idea.

Comment (ali-ramadhan) [inline Makefile]: Nice echo.

Comment (basimr) [inline Makefile]: The -v flag was too verbose (I know...) I added some echo statements though.

**Pull request merged**